releasenotes/notes/profiling-local-root-span-fd7ee6a08b8612b7.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
```

Comment:
this is a bug fix right?

Reply:
what changes in the UI/public facing from this change?

just now wondering if this actually needs a changelog entry or not.